### ISSUE TITLE
notify event not trigger

### ISSUE THREAD

**Issue by manzooralam** [cipchk/ngx-countdown#94]
I used this library, but notify event not happened?. 

TemplateCode: 
` <countdown [config]="{leftTime: 10,
    format: 'mm:ss', notify: 5}"
     (event)="handleEvent($event)"></countdown>`

handleEvent method code:
```
handleEvent(event: CountdownEvent): void {
    // notify action not trigger
     console.log(event);
    if (event.action === 'notify') {
      console.log(event);
    }
}
```
[Here stackblitz code  ](https://stackblitz.com/edit/ngx-countdown-notify?file=src/app/app.component.ts),

**Comment by cipchk** [comment]
`notify` is `number[]` type when you don’t need to trigger every second.

**Comment by cipchk** [comment]
fixed by `11.0.3`
